components/homme/src/share/gllfvremap_util_mod.F90

```diff
@@ -536,6 +591,8 @@ function gfr_check_api(hybrid, nets, nete, hvcoord, elem) result(nerr)
     integer :: nphys, ftype_in, ftype_idx, boost_idx, nerr
     logical :: boost_pg1
 
+    is_sphere = trim(geometry) /= 'plane'
```

Comment:
unrelated to this code, but why do we have both geometry and topology in control_mod?
```
if (topology == "cube") then
       write(iulog,*)"readnl: ne,np         = ",ne,np
else if (topology == "plane") then
  write(iulog,*)"readnl: ne_x,ne_y,np         = ",ne_x,ne_y,np
end if
```

Reply:
Topology is for the grid, geometry for the metric terms. Chris did that.